.gitignore

```diff
@@ -30,6 +30,12 @@ cmd/system-shutdown/unit-tests
 # manual pages
 cmd/*/*.[1-9]
 
+# auto-generated systemd units
+data/systemd/snapd.autoimport.service
```

Comment:
You could change this to be `data/systemd/*.service` now, so that you don't have to keep updating this..